manifests/config.pp

```diff
@@ -157,66 +157,58 @@
     # the vast majority of files here are init-files
     # so any change there should trigger a full service restart
     if $prometheus::server::restart_on_change {
-      File {
-        notify => Class['prometheus::run_service'],
-      }
-      $systemd_notify = [Exec['prometheus-systemd-reload'], Class['prometheus::run_service']]
```

Comment:
Both this variable and the `Exec` resource disappeared 2 years ago in https://github.com/voxpupuli/puppet-prometheus/pull/90